packages/realm/bindgen/vendor/bindgen-lib/package.json

```diff
@@ -6,8 +6,10 @@
   "bin": {
     "realm-bindgen": "./realm-bindgen.ts"
   },
+  "exports": {
+      "./*": "./src/*"
```

Comment:
I'd prefer if we didn't declare the entire internal implementation of the bindgen package as public / exported to the SDK specific code. Instead I'd suggest we export a single `index.ts` which exports the parts we actually want exposed. This will help us iterate the binding generator codebase without breaking the generators between SDKs.

Reply:
I'm fine with that as a follow-up, but I don't want to do that in this commit. I'm not 100% sure what we want to be public, but I suspect it is close to 90% of the code, so it may not be worth making a fine-grained distinction here.

Reply:
> not 100% sure what we want to be public

We don't have to be, for now we should just export whatever the JS generators need. Now, you're effectively declaring everything public 🤷

Reply:
If 90% of the code is public (my rough guess), I think the benefit from making the remaining 10% private is fairly low. The main benefit IMO is that it would let you have a single import in the consumer rather than several.